receiver/iisreceiver/scraper_test.go

```diff
@@ -162,10 +164,10 @@ type mockPerfCounter struct {
 	value    float64
 }
 
-func newMockWatcherFactory(watchErr error, value float64) func(string, string,
+func newMockWatcherFactory(watchErr error) func(string, string,
```

Comment:
Is this change something requested by the linter, or are you just removing the input argument since it's currently always `1`?

Reply:
It was spotted by the linter, but, what it complains is that it is always 1 and suggests removing the parameter.

Reply:
That's fair. The argument can always be added back in if someone wants to pass in a different value.